### PR TITLE
Rc 350

### PR DESCRIPTION
**Pull request by prckent**
Release candidate for v3.5.0

@jtkrogel Please check CHANGELOG.md for NEXUS updates

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,59 @@
...
+QMCPACK is ongoing. Track the develop branch and follow discussion on
+GitHub closely to avoid difficult merges.
+
+* Support for Quantum Espresso 6.3 and 6.2.1. Check documentation to
+  ensure compiled with required HDF5 support.
+* Support for periodic gaussians and PySCF generated
+  wavefunctions. Initial version is limited to Gamma-point.
+* Improved hybrid representation of single particle orbitals
+  (APW-like) for significantly reduced memory usage and possible
+  accuracy increase compared to conventional spline representation.
```
Comment (ye-luo) [inline CHANGELOG.md]: Could you add an arxiv pointer here? arXiv:1805.07406

Comment (prckent) [reply CHANGELOG.md]: Done

**Comment by prckent** [comment]
@jtkrogel confirms updated CHANGELOG is OK on Slack

**Comment by prckent** [comment]
OK to test

**Pull request merged**